manuscript/Cycle-Notation.md

```diff
@@ -29,7 +29,7 @@ leave out numbers that are not permuted. In our example we end up with the
 following cycle notation.
 
 ```plain
-(0 1 2)(3 4)
+(0 2 1)(3 4)
 ```
```

Comment:
True, the text states that, but the example above it shows:
```
[
    0 1 2 3 4 5
    1 2 0 4 3 5
]
```
..in which `0` goes to `2`, `2` goes to `0` and `1` goes `1`. 

I'm confused, and probably not understanding this correctly..